src/Xamarin.Android.Build.Tasks/Tasks/JavaToolTask.cs

```diff
@@ -78,6 +78,8 @@ at com.android.dx.command.Main.main(Main.java:106)
 
 		public string JavaMaximumHeapSize { get; set; }
 
+		public virtual string DefaultErrorCode => null;
```

Comment:
@jonathanpeppers, @brendanzagaeski  we have a bunch of tooling which derive from `JavaToolTask` and will probably us this right? We should probably put in a work item/issue to make sure they all have proper MSBuild Error codes.